assets/js/googlesitekit/datastore/user/feature-tours.js

```diff
@@ -153,6 +191,13 @@ const baseControls = {
 
 		return true;
 	} ),
+	[ CACHE_LAST_DISMISSED_AT ]: async ( { payload } ) => {
+		const { timestamp } = payload;
+
+		await setItem( FEATURE_TOUR_CACHE_KEY, timestamp, {
+			ttl: FEATURE_TOUR_COOLDOWN_PERIOD,
```

Comment:
The `ttl` here is expected to be a number in seconds, however `FEATURE_TOUR_COOLDOWN_PERIOD` is defined in ms (`1000 * ...`), so this would result in a cooldown 1000x longer than we want 😄 

It would be good to rename `FEATURE_TOUR_COOLDOWN_PERIOD` to `FEATURE_TOUR_COOLDOWN_SECONDS` to clarify the unit.

Reply:
Oops 😂
Yes, good idea!